doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -19,6 +19,7 @@
     * [3.2.2 Container Networking Services](#322-container-networking-services)
     * [3.2.3 Container Storage Services](#323-container-storage-services)
     * [3.2.4 Container Package Managers](#324-container-package-managers)
+        * [3.2.5.1 Operator Pattern](#3251-operator-pattern)
```

Comment:
Typo: 3.2.4.1 instead of 3.2.5.1

Reply:
@pgoyal01 : Actually it doesn't logically fit under 3.2.4 so I changed it into its own 3.2.5.